modules/luci-mod-dsl/Makefile

```
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2022 Roland Barenbrug <roland@treslong.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LUCI DSL spectrum graph
+LUCI_DEPENDS:=@TARGET_lantiq_xrx200 +luci-base +ltq-dsl-base
```

In the meantime there's https://github.com/openwrt/openwrt/commit/1afd0fefd22992b808e98fcf726a8e4d3af91604 , so please use:
```suggestion
LUCI_DEPENDS:=+luci-base ltq-dsl-app
```